### PR TITLE
Bug in from_didl_string causes incomplete information to be returned by soco.get_queue()

### LINKED ISSUE #552: Pull request 455 breaks event object for Soundcloud
With the new event handling functionality merged I updated my sonoscontroller to use the new version. Apparently this pull request changes the data returned in an avevent. I have the following code

`
metadata = event.current_track_meta_data 
metadata.album_art_uri
`
After the pull request the event is now returning a MSTrack instead of a DidlMusicTrack for metadata which only seems to contain the title. Is this intentional and should the MSTrack object contain the all the relevant information or is this a bug? Spotify on the other hand still returns a DidlMusicTrack.

### PR DESCRIPTION
**Pull request by relevitt**
One line of code should be commented out until it has been adequately tested and is bug free.
Closes #552.

### CODE REVIEW

File: soco/data_structures_entry.py [hunk 1]
```diff
@@ -50,7 +50,10 @@ def from_didl_string(string):
             except KeyError:
                 raise DIDLMetadataError("Unknown UPnP class: %s" % item_class)
             item = cls.from_element(elt)
-            item = attempt_datastructure_upgrade(item)
+            # The following line has been commented out because
+            # it results in soco.get_queue() returning incomplete
+            # information and likely has other unintended consequences
+            #item = attempt_datastructure_upgrade(item)
```
Comment (ghcs27) [inline soco/data_structures_entry.py]: PEP8 complains: Please add a space after "#".

Comment (relevitt) [reply soco/data_structures_entry.py]: @ghcs27 thank you. Space added.

**Comment by KennethNielsen** [comment]
I have made an alternative PR in #589 which retains all the metadata while keeping the appropriate data structure.

**Comment by KennethNielsen** [comment]
I've merged the alternative PR #589 thereby closing this

**Pull request closed without merging**